src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -688,12 +690,15 @@ default boolean canDisableShield(ItemStack stack, ItemStack shield, LivingEntity
     }
 
     /**
-     * Is this Item a shield
+     * {@return {@code true} if this item is considered a shield}
      *
-     * @param stack  The ItemStack
-     * @param entity The Entity holding the ItemStack
-     * @return True if the ItemStack is considered a shield
+     * @param stack  the item stack
+     * @param entity the entity holding the item stack
+     * @deprecated To be removed in 1.18. Override {@link #canPerformAction(ItemStack, ToolAction)} and return
+     * {@code true} if the passed in tool action is contained in {@link ToolActions#DEFAULT_SHOVEL_ACTIONS} or is
```

Comment:
Should be default shield actions instead of default shovle actions.

Reply:
Done. _(how did I miss this?!)_